apypie/api.py

```diff
@@ -60,7 +62,34 @@ def __init__(self, **kwargs):
         if kwargs.get('username') and kwargs.get('password'):
             self._session.auth = (kwargs['username'], kwargs['password'])
 
-        self.apidoc = self._load_apidoc()
+        self._apidoc = self._load_apidoc()
```

Comment:
Why call it here? It is already lazily loaded in line 70.

Reply:
mostly because that required tests adjustments and I was lazy?
fixed in c1bf52b